site/content/learn/headless/basics-performance.md

```diff
@@ -92,13 +108,102 @@ The Resource Timing API allows us to zoom in to single resources and get accurat
 {{< /tab >}}
 {{< /tabs >}}
 
+#### Paint Timing API (`first-paint` and `first-contentful-paint`)
+
+[The Paint Timing API](https://developer.mozilla.org/en-US/docs/Web/API/PerformancePaintTiming) provides information on the first paint and the first contentful paint. Access the entries via `performance.getEntriesByType('paint')` or `performance.getEntriesByName('first-contentful-paint')`.
+
+{{< tabs "3" >}}
```

Comment:
@stefanjudis I am not sure I am sold on having the output of the API commented in the code. On one hand it saves space, but on the other I feel it takes away from clarity and tidiness. Do we want to just add a line below with something like "Here is the output:" followed by a code block with the output from the console.log?

Reply:
As discussed in Slack I'll introduce a new `details` block. 

<img width="902" alt="image" src="https://user-images.githubusercontent.com/962099/174992431-61a607a7-2887-406d-94c9-b0ee95ace41e.png">

<img width="963" alt="image" src="https://user-images.githubusercontent.com/962099/174992517-09a5d444-6f6c-436a-bd5d-78c94028d736.png">